src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/QuestionPriceArea.java

```diff
@@ -0,0 +1,106 @@
+package io.github.oliviercailloux.y2018.apartments.valuefunction.profile;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.collect.Range;
+import io.github.oliviercailloux.y2018.apartments.valuefunction.Criterion;
+
+public class QuestionPriceArea {
+  int price;
+  int surface;
+  private final String question = "Would you pay %d€ more for %d m2 more?";
+
+  /**
+   * Instantiates a new question price/area.
+   *
+   * @param price the price shown to the user according to his/her profile
+   * @param surface the surface shown to the user according to his/her profile
+   */
+  private QuestionPriceArea(int price, int surface) {
+    this.price = price;
+    this.surface = surface;
+  }
+
+  /**
+   * Creates the object question.
+   *
+   * @param price the price
+   * @param surface the surface
+   * @return the question price area
+   */
+  public static QuestionPriceArea create(int price, int surface) {
+    return new QuestionPriceArea(price, surface);
+  }
+
+  /**
+   * Getter for attribute price
+   *
+   * @return the price : question parameter
+   */
+  public int getPrice() {
+    return this.price;
+  }
+
+  /**
+   * Getter for attribute surface
```

Comment:
Unité

Reply:
Fix in 7c916915ee4778efa9494da1165860554ec313bc